app/shared/src/__tests__/GestureController.test.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+import GestureController from '../GestureController';
+
+describe('GestureController', () => {
+  it('should call NativeModules.RNKiwiGestureController.addGestures', () => {
+    GestureController.addGestures('KiwiHotels');
+    expect(
+      NativeModules.RNKiwiGestureController.addGestures,
+    ).toHaveBeenCalledWith('KiwiHotels');
+  });
+});
```

Comment:
I think what is more important to test (and nice) would be to make sure that `noop` is called when there is no native module present.

To test it, we should probably not mock `NativeModules.RNKiwiGestureController` in one suite.